src/components/Routes.tsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react';
 import { Route } from 'react-router-dom';
+import Users from '../components/frontTestRoads/user/Users';
+import User from './frontTestRoads/user/User';
+import Companies from './company/Companies';
+import Company from './company/Company';
+import Professions from './professions/Professions';
+import ProjectList from './project/ProjectList';
 import HomePage from '../views/HomePage';
 import SuperAdmin from './home/SuperAdmin';
 import ExportRecords from './records/Exporter/ExportRecords';
 import ListsCompanies from './companies/ListsCompanies';
 import FormResult from './records/Exporter/FormResult';
+import DetailsProjects from './company/DetailsProjects';
 
 function Routes(): JSX.Element {
   return (
     <>
       <Route exact path="/aeviso" component={SuperAdmin} />
       <Route exact path="/home" component={HomePage} />
       <Route exact path="/clients" component={ListsCompanies} />
+      <Route exact path="/companies" component={Companies} />
+      <Route exact path="/companies/:id" component={Company} />
+      <Route exact path="/companies/*/projects/:id" component={DetailsProjects} />
```

Comment:
`*` ?

Reply:
The `*` here works the same way as on Express, here we don't need the company's id so we used this instead

Reply:
So why not just `/project/:id` ?

Reply:
There has been a problem of logic on the way, this will be fixed

Reply:
fixed !